### PR TITLE
Compatibility changes for the pre-1.0.0 version of the client

### PR DESCRIPTION
**Pull request by twolodzko**
Nothing else than `stringify_unsupported` seems to be needed here.

### CODE REVIEW

**Review by kshitij12345 — CHANGES_REQUESTED** [review]
Seems like I commented on other PR about `stringify_unsupported` 😓 

In any case, I think the docstrings should be updated to not use `neptune.init()`.

**Comment by twolodzko** [comment]
What do you mean by `stringify_unsupported` here @kshitij12345 ?

File: tests/test_e2e.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    y = iris.target
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5)
+
+    model = LogisticRegression(C=1e5)
+    model.fit(X_train, y_train)
+
+    run["summary"] = npt_utils.create_classifier_summary(model, X_train, X_test, y_train, y_test)
+
+    run.wait()
+    validate_run(run, log_charts=True)
```
Comment (kshitij12345) [inline tests/test_e2e.py]: Should we also test with `log_charts=False`?

Comment (twolodzko) [reply tests/test_e2e.py]: The tests are right now very minimalistic, basically a smoke test. Definitely, it'd benefit to improve them, but I'm not sure if this is in scope.

Comment (kshitij12345) [reply tests/test_e2e.py]: Fair enough.

**Review by kshitij12345 — APPROVED** [review]
LGTM! Thank you @twolodzko

**Pull request merged**